webapp/platform/components/src/generic_modal/generic_modal.tsx

```diff
@@ -55,6 +56,13 @@ export type Props = {
     showCloseButton?: boolean;
     showHeader?: boolean;
 
+    /**
+     * Whether this modal is stacked on top of another modal.
+     * When true, the modal will not render its own backdrop and will
+     * adjust the z-index of the parent modal's backdrop.
+     */
+    isStacked?: boolean;
```

Comment:
So in order to make this work, do we have to make sure this property is set?
If so, I'm wondering how best to communicate this to developers so they know to do it.